CMakeLists.txt

```diff
@@ -146,6 +148,14 @@ SET(MUSESCORE_NAME_VERSION "${MUSESCORE_NAME} ${MUSESCORE_VERSION_MAJOR}")
 #
 SET(MUSESCORE_VERSION_FULL  "${MUSESCORE_VERSION}.${MUSESCORE_VERSION_PATCH}")
 
+if (CMAKE_MUSESCORE_LABEL)
```

Comment:
CMAKE_SKIP_RPATH or CMAKE_BUILD_WITH_INSTALL_RPATH are documented CMake variables but CMAKE_MUSESCORE_LABEL is a completely new variable. I would use a different "namespace" for this. MUSESCORE_LABEL ?